README.md

```diff
@@ -1,7 +1,11 @@
+<!--BEGIN_BANNER_IMAGE--><!--END_BANNER_IMAGE-->
+
 # LiveKit: Real-time video, audio and data for developers
 
+<!--BEGIN_DESCRIPTION-->
 [LiveKit](https://livekit.io) is an open source project that provides scalable, multi-user conferencing based on WebRTC.
 It's designed to provide everything you need to build real-time video/audio/data capabilities in your applications.
+<!--END_DESCRIPTION-->
```

Comment:
Actually, for now let's keep the first paragraph of this repo unmanaged — it's kind of special as many users land here if, eg, they just search "livekit".

Reply:
The full description is quite unique compared to the other repos. I would suggest copying and pasting the full text into the repo manager to overwrite the default server sdk description. That way we don't end up with a partially maintained description text which could lead to unexpected descriptions in the future.
How does that sound to you?

Reply:
Yeah that sounds like the right thing to do